drivers/volume/portworx/portworx.go

```diff
@@ -2461,29 +2467,37 @@ func (d *portworx) DecommissionNode(n *node.Node) error {
 		}
 	}
 
-	if err := d.StopDriver([]node.Node{*n}, false, nil); err != nil {
+	if err := d.EnterMaintenance(*n); err != nil {
 		return &ErrFailedToDecommissionNode{
 			Node:  n.Name,
 			Cause: fmt.Sprintf("Failed to stop driver on node: %v. Err: %v", n.Name, err),
 		}
 	}
 
+	time.Sleep(10 * time.Second)
```

Comment:
why code is sleep for 10 second here? Can we replace sleep with wait method to make sure node went into maintenance mode...

Reply:
sleep not required here.Fixed it